cmd/cloud-controller-manager/OWNERS

```diff
@@ -0,0 +1,13 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- aojea
+- justinsb
+- sdmodi
```

Comment:
s/sdmodi/bowei/

sudeep is no longer at google

Reply:
Fixed.